docs/nydusd.md

```diff
@@ -256,6 +256,9 @@ Currently, the mirror mode is only tested in the registry backend, and in theory
           {
             // Mirror server URL (including scheme), e.g. Dragonfly dfdaemon server URL
             "host": "http://dragonfly1.io:65001",
+            // true: authorization from origin registry and through mirror, e.g. Using Dragonfly server as mirror.
```

Comment:
We can revise the comment description a bit:

// true:  Send the authorization request to the mirror e.g. Dragonfly.
// false:  Authorization request won't be relayed by the mirror e.g. another docker registry.

Reply:
Sorry, already fixed it.